### ISSUE TITLE
NpmPackageManager doesn't load nested dependecies

### ISSUE THREAD

**Issue by ktarasenko** [cqframework/clinical_quality_language#847]
My assumption, that NpmPackageManager will pick up nested dependencies from `package.json` files of its dependencies. 

Instead, it only loads `hl7.fhir.r4.core` and everything the `sourceIg` is depending on without going deeper. 

I added a test to illustrate it in the branch 
https://github.com/ktarasenko/clinical_quality_language/tree/nested_dependencies
```
./gradlew :cqf-fhir-npm:test --tests "org.cqframework.fhir.npm.NpmPackageManagerTests.TestNestedDependencies"
```

Expected dependency structure: 
```
test.source
-- test.dep1
---- test.dep3
-- test.dep2
---- test.dep4
```

**Comment by brynrhodes** [comment]
Came up on WHO call this morning, committed to providing an update first meeting of the new year (Jan 9)

**Comment by c-schuler** [comment]
Resolved in #1149
